### ISSUE TITLE
Getting linked Framework error while testing appium-webdriveragent

### ISSUE THREAD

**Issue by KiratiBhuva** [appium/appium#14145]
## The problem

When I try to test with a real device and scheme WebDriverAgentRunner, It is giving the below error. 
It persists in Appium as well as Appium-desktop. 

Building for iOS, but the linked framework ‘CocoaAsyncSocket.framework’ was built for tvOS + tvOS Simulator. (in target ‘WebDriverAgentLib’ from project ‘WebDriverAgent’

![image](https://user-images.githubusercontent.com/35611972/78404183-b46ea500-75b2-11ea-82de-e3c643ca0732.png)

## Environment

* Appium version (or git revision) that exhibits the issue: 1.15.1
* Desktop OS/version used to run Appium: macOS Catalina - 10.15.4
* Mobile platform/version under test: iPhone 8 Plus - 13.3 
* Xcode version - 11.4

**Comment by KiratiBhuva** [comment]
Appium server 1.15.1-beta seems to be working. So, If I need to use appium-desktop, I need to use custom server 1.15.1-beta with appium-desktop 1.15.1.

Please correct me if I am wrong

**Comment by mykola-mokhnach** [comment]
Duplicate of https://github.com/appium/appium/issues/14102

**Comment by Whiperdief** [comment]
Same issue. None of the above helps. (iOS)

Reinstalled appium-desktop.
Reinstalled appium CLI.
Reinstalled node.
Removed everything appium/node related.
Run appium-doctor, no errors or warning. 

I did find a "solution" to my problem.
When I execute JAVA code testing following folder is generated:
... Library/Developer/Xcode/DerivedData/WebDriverAgent-ciegwgvxzxdrqthilmrmczmqvrgu

When I execute appium-desktop following folder is generated:
... Library/Developer/Xcode/DerivedData/WebDriverAgent-akxhdiizyykdxefsthlnfyomlaiz

The CLI appium generated folder has  folder named 'build'. Copy paste that into the folder generated by appium-dekstop.
